### PR TITLE
ENG-3456 Improved log message for resource not found

### PR DESCRIPTION
**Pull request by entgigi**

### CODE REVIEW

File: src/main/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllers.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public HasMetadata loadStandardResource(String kind, String namespace, String na
                         .fromServer()
                         .get())
                 .orElseThrow(() -> new IllegalStateException(
-                        "Resource kind '" + kind + "' not supported."));
+                        String.format("Unable to load resource '%s/%s/%s'", kind, namespace, name)));
```
Comment (user-54dbb57a) [inline src/main/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllers.java]: @entgigi 

```
Unable to load resource "%s" of kind "%s" from namespace "%s"
```

File: src/test/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllersTest.java [hunk 2]
```diff
@@ -315,12 +315,31 @@ void shouldRetrieveStandardResourceGenerically() throws InterruptedException {
         step("And the same applies for Deployments, Services, Ingresses, Secrets and PersistentVolumeClaims", () -> {
             assertThat(SupportedStandardResourceKind.DEPLOYMENT.getOperation(getFabric8Client()))
                     .isInstanceOf(DeploymentOperationsImpl.class);
-            assertThat(SupportedStandardResourceKind.SERVICE.getOperation(getFabric8Client())).isInstanceOf(ServiceOperationsImpl.class);
```
Comment (user-54dbb57a) [inline src/test/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllersTest.java]: @entgigi if it's just reformat please remove it from the PR

File: src/test/java/org/entando/kubernetes/controller/support/creators/IngressCreatorTest.java [hunk 3]
```diff
@@ -132,7 +133,8 @@ void shouldTruncateTheHostNameIfRequired() {
         creator.createIngress(client.ingresses(), deployable, serviceCreator.getService(),
                 new ServerStatus(NameUtils.MAIN_QUALIFIER));
 
-        String expectedHost = "my-app-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-my-app-namespace.apps.autotest.eng-entando.com";
+        String hostName = Optional.ofNullable(System.getenv("ENTANDO_DEFAULT_ROUTING_SUFFIX")).orElse("somehost.com");
```
Comment (user-54dbb57a) [inline src/test/java/org/entando/kubernetes/controller/support/creators/IngressCreatorTest.java:136]: off topic?

**Review by user-54dbb57a — CHANGES_REQUESTED** [review]

File: src/test/java/org/entando/kubernetes/controller/support/creators/IngressCreatorTest.java (hunk 3 above)
Comment (entgigi) [reply src/test/java/org/entando/kubernetes/controller/support/creators/IngressCreatorTest.java:136]: it isn't, it doesn't work without it

File: src/main/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllers.java (hunk 1 above)
Comment (entgigi) [reply src/main/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllers.java]: done

File: src/test/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllersTest.java (hunk 2 above)
Comment (entgigi) [reply src/test/java/org/entando/kubernetes/controller/spi/client/impl/DefaultKubernetesClientForControllersTest.java]: removed

**Review by user-54dbb57a — APPROVED** [review]

**Pull request merged**